src/formio/components/Map.js

```diff
@@ -90,16 +91,26 @@ export default class Map extends Field {
     this.setValue(newLatLng, {modified: true});
   }
 
+  setAndReturnCoordinates(coordinates) {
+    this.onMarkerSet(coordinates);
+    return coordinates;
+  }
+
   renderReact() {
-    const markerCoordinates = this.getValue();
+    const startingPop = [this.component.lat, this.component.lng] || DEFAULT_LAT_LON;
+    const markerCoordinates =
+      this.getValue().length === 2 ? this.getValue() : this.setAndReturnCoordinates(startingPop);
+
     const container = this.refs.mapContainer;
+    const zoom = Number(this.component.defaultZoom);
     // no container node ready (yet), defer to next render cycle
     if (!container) return;
 
     this.reactRoot.render(
       <LeafletMap
-        markerCoordinates={markerCoordinates || null}
+        markerCoordinates={markerCoordinates}
         onMarkerSet={this.onMarkerSet.bind(this)}
+        zoomLevel={zoom || MAP_DEFAULTS.zoom}
       />
     );
```

Comment:
this isn't correct - the marker implies the select location and when no default value is provided, there just initially are no coordinates selected. Therefor, `markerCoordinates` can still be `null`.

It looks like you're trying to use `markerCoordinates` to set the center of the map view, but that's an entirely separate concept/prop that should be added.